src/lib.rs

```diff
@@ -864,7 +864,9 @@ impl DubinsPath {
     #[must_use]
     pub fn sample_many(&self, step_distance: FloatType) -> Vec<PosRot> {
         let zero: FloatType = 0.0;
-        self.sample_many_range(step_distance, zero..self.length())
+        let mut samples = self.sample_many_range(step_distance, zero..self.length());
```

Comment:
I think there is a better way to handle this, which is to have `0..=self.length()` and to modify `sample_many_range` to make that behave correctly. However, I can do that later and will accept your PR

Reply:
As in make `sample_many_range` include the endpoint in all cases? ie: do integer steps until the last one comes up, which is an FP based step?